pkg/plugins/autodiscovery/flux/utils.go

```diff
@@ -0,0 +1,78 @@
+package flux
+
+import (
+	"fmt"
+	"io/fs"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+// searchFluxFiles will look, recursively, for every files containing a flux helmrelease or helmrepository from a root directory.
+func (f *Flux) searchFluxFiles(rootDir string, files []string) error {
+
+	err := filepath.Walk(rootDir, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			fmt.Printf("prevent panic by handling failure accessing a path %q: %v\n", path, err)
```

Comment:
Should this be a log rather than `fmt.Printf`?

Reply:
Thanks that's a good catch